### DISCUSSION TITLE
Discuss which kind of features/packages should be built in.

### DISCUSSION THREAD

**Discussion by harryfei** [emacs-ng/emacs-ng#472] [Ideas]
We are trying to make more and more features/packages built in.
Such as

* default melpa packages https://github.com/emacs-ng/emacs-ng/issues/98
* libegit2 https://github.com/emacs-ng/emacs-ng/issues/100

IMHO, we should have a discussion about which kind of features/packages should be built in and which kind not.

**Comment by brotzeit** [comment]
Regarding lisp packages #98 should be enough. I think rust features should be configured optionally like webrender.

Package management and git support are the most basic features that everybody uses, so it makes sense to improve them by default.

**Comment by DavidDeSimone** [comment]
I think we should lock in the following features as our major goal (not in any order):

* Deno/JS
* WebRender
* Package Management Improvements
* Git Support

Not counting performance improvements, since I don't consider that a "feature" for this purpose. If someone comes up with an awesome improvement not aligned with this list we can have an additional discussion on if it should be included. However, for now, I propose we make those 4 things into our next major milestone.
